src/Diagnostics/FxCop/CSharp/Microsoft.CodeAnalysis.CSharp.FxCopAnalyzers.props

```diff
@@ -8,7 +8,6 @@
   <PropertyGroup>
     <CodeAnalysisRuleSetOverrides>$(CodeAnalysisRuleSetOverrides);
       -Microsoft.Design#CA1003;
-      -Microsoft.Design#CA1019;
```

Comment:
Can this kind of stuff needs to be live?


-          heejae

From: Manish Vasani [mailto:notifications@github.com]
Sent: Wednesday, March 18, 2015 4:06 PM
To: dotnet/roslyn
Cc: HeeJae Chang
Subject: Re: [roslyn] Create props file in the System.Runtime.Analyzers with CodeAnalysisOverr... (#1369)


In src/Diagnostics/FxCop/CSharp/Microsoft.CodeAnalysis.CSharp.FxCopAnalyzers.props<https://github.com/dotnet/roslyn/pull/1369#discussion_r26716680>:

> @@ -8,7 +8,6 @@

>    <PropertyGroup>

>      <CodeAnalysisRuleSetOverrides>$(CodeAnalysisRuleSetOverrides);

>        -Microsoft.Design#CA1003;

> -      -Microsoft.Design#CA1019;

We should probably write an analyzer to verify that the props file always stays up-to-date:

  1.  It has no extra entries for IDs that are not part of any analyzer in owing project.
  2.  Every newly added CA analyzer has an entry in here.

Logically, an equivalent of our DeclarePublicAPI analyzer. This would be helpful as we port more and more rules to FXCop analyzer projects.

—
Reply to this email directly or view it on GitHub<https://github.com/dotnet/roslyn/pull/1369/files#r26716680>.